### PR TITLE
Added filtered functionality in download button.

### PR DESCRIPTION
**Pull request by naseemshah**
Now download button downloads the filtered results

### CODE REVIEW

File: src/Components/Common/PrescriptionBuilder.gen.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import * as React from 'react';
 // tslint:disable-next-line:no-var-requires
 const PrescriptionBuilderBS = require('./PrescriptionBuilder.bs');
 
-import {t as Prescription__Prescription_t} from '@coronasafe/prescription-builder/src/Types/Prescription__Prescription.gen';
+import {t as Prescription__Prescription_t} from './Prescription__Prescription.gen';
```
Comment (bodhish) [inline src/Components/Common/PrescriptionBuilder.gen.tsx]: Why did you change this 🤔

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Common/PrescriptionBuilder.gen.tsx (hunk 1 above)
Comment (naseemshah) [reply src/Components/Common/PrescriptionBuilder.gen.tsx]: I didn't, it automatically happened. I thought it was part of some scripts.

File: src/Components/Patient/ManagePatients.tsx [hunk 2]
```diff
@@ -98,6 +98,8 @@ export const PatientManager = (props: any) => {
     const params = {
       csv: true,
       facility: facilityId,
+      disease_status: qParams.disease_status
```
Comment (SandeepThomas) [inline src/Components/Patient/ManagePatients.tsx:101]: No need for name and phone_number ? @bodhish  @tomahawk-pilot

Comment (bodhish) [reply src/Components/Patient/ManagePatients.tsx:101]: It's more like a search than a filter function. I guess we could avoid it for now.

**Comment by dauntlessnomad** [comment]
@SandeepThomas  please merge if this is clean

File: src/Components/Common/PrescriptionBuilder.gen.tsx (hunk 1 above)
Comment (naseemshah) [reply src/Components/Common/PrescriptionBuilder.gen.tsx]: so should i change it back?

Comment (bodhish) [reply src/Components/Common/PrescriptionBuilder.gen.tsx]: Yes please

Comment (naseemshah) [reply src/Components/Common/PrescriptionBuilder.gen.tsx]: rolled the changes back. see commit 2d7304c

**Review by bodhish — APPROVED** [review]
